src/components/grouped-character/__spec__.js

```diff
@@ -62,6 +62,16 @@ describe('GroupedCharacter', () => {
         expect(input.nodes[0].value).toEqual('12/3456/789');
       });
     });
+
+    describe('on initial render when a value is passed in', () => {
+      it('renders the value with separators', () => {
+        wrapper = mount(
+          <GroupedCharacter onChange={ jasmine.createSpy('onChange') } groups={ [2, 2, 2] } value='123456' />
```

Comment:
Prefer `jest.fn()` over `jasmine.createSpy`.

Reply:
do you even need the `onChange` prop?

Reply:
or even just a function `() => {}`

Reply:
Don't need it, was just copied from previous test.